src/workflows/airqo_etl_utils/data_api.py

```
@@ -856,15 +898,135 @@
+            measurements = pd.DataFrame(
+                columns=["value", "variable", "station", "time"]
+            )
+
+        return measurements
 
-    def __request(
+    # TODO Make this private after centralizing data ops
+    def _request(
         self,
         endpoint: str,
         params: Optional[Dict[str, Any]] = {},
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)